### ISSUE TITLE
Advanced Search Feature Request: filter by groups of transactions

### ISSUE THREAD

**Issue by mvadari** [xrpscan/xrpscan.com#47]
It'd be nice if e.g. instead of entering in all the AMM transaction names in an OR group, you could just have that group of transactions as a filter.

This could perhaps also be achieved by a "contains" filter on `TransactionType`.

**Comment by pkcs8** [comment]
Yes, switching to multi select `TransactionType` would be useful. I'll take a look at querybuilder docs to see if it can do this.
